src/libraries/System.Threading.Channels/tests/System.Threading.Channels.Tests.csproj

```
@@ -6,19 +6,17 @@
   <ItemGroup>
     <Compile Include="BoundedChannelTests.cs" />
     <Compile Include="ChannelClosedExceptionTests.cs" />
+    <Compile Include="ChannelClosedExceptionTests.netcoreapp.cs"
+             Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'" />
```

tiny nit: This is fine today as we only build our test assemblies against the latest .NET version but we usually avoid conditioning on a specific TFM unless it makes sense in that context.

```suggestion
             Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" />
```